mod-source-record-manager-server/src/main/java/org/folio/dao/util/JobExecutionDBConstants.java

```diff
@@ -80,6 +82,11 @@ public final class JobExecutionDBConstants {
     "LEFT JOIN total p ON true " +
     "LIMIT $1 OFFSET $2";
 
+  public static final String GET_UNIQUE_USERS = "WITH unique_users AS (SELECT DISTINCT user_id, " +
+    "job_user_first_name, job_user_last_name FROM %s)," +
+    "total AS (SELECT count(*) AS total_count FROM unique_users)" +
+    "SELECT j.*, p.* FROM unique_users j LEFT JOIN total p ON true LIMIT $1 OFFSET $2";
+
```

Comment:
Seems it also required to filter by is_hidden field here, because delete authority job uses hidden profile, and user for it should not be provided in response

Reply:
Hi @SerhiiNosko! Do you mean 'is_deleted'? I added a condition for it.

Reply:
I mean job_profile_hidden, the same as in related endpoint for another dropdown:
https://github.com/folio-org/mod-source-record-manager/commit/e1b2bf7ecf2e4ed8f48f325546db1c68f64c9656#diff-426ed7c82b47e1e393acd421afd7c05b73d48f13cdb3e2fae4afb5d024b72f10R76

Reply:
@SerhiiNosko I added new condition!